openid-sharedsignals-framework-1_0.md

```diff
@@ -743,16 +743,7 @@ Transmitters have metadata describing their configuration:
 
 spec_version
 
-> OPTIONAL. A version identifying the implementer's draft or final specification implemented by the Transmitter. This includes the numerical portion of the spec version as described in the document {{NAMINGCONVENTION}}. If absent, the Transmitter is assumed to conform to "1_0-ID1" version of the specification (this document).
-
->  The following is a non-normative example of a Transmitter that implements the third implementer's draft of the Shared Signals Framework specification 1_0.
-
-~~~ json
-   {
-        "spec_version": "1_0-ID3"
-   }
-~~~
-{: #figspecversionid2 title="Example spec_version referring to the 2nd implementer's draft of the 1_0 spec"}
```

Comment:
Not sure why we are removing this example. I thought we would keep it there, just update the previous line 755 to say "...3rd implementer's draft" to match the example.

Reply:
The ID3 spec version will still have it, I thought it would be better to clean up from v1 spec.